avm/src/lib.rs

```diff
@@ -70,8 +70,24 @@ pub fn use_version(version: &Version) -> Result<()> {
     Ok(())
 }
 
+/// Update to latest stable version
+pub fn update() -> Result<()> {
+    // Find last stable version
+    let version = &get_latest_version();
+
+    install_version(version)?;
+    Ok(())
```

Comment:
nit: you can do `install_version` as the last line in the function.

same with `use_version` below

Reply:
done